openstack_taster.gemspec

```diff
@@ -3,7 +3,14 @@
 Gem::Specification.new do |spec|
   spec.name        = 'openstack_taster'
   spec.version     = '0.0.1'
-  spec.summary     = 'Taste all of the OpenStacks'
+  spec.summary     = "Taste all of the OpenStack's basic functionality for an image"
+  spec.description = "Tastes images on an OpenStack deployment for security and basic usability"
   spec.author      = ['OSU Open Source Lab']
   spec.files       = ['lib/openstack_taster.rb']
```

Comment:
You need to add ``spec.executables << 'openstack_taster'`` here so that it can install the executable you've created.